airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -986,7 +987,10 @@ void testSyncUsesAirbyteStreamNamespaceIfNotNull() throws Exception {
         Jsons.deserialize(
             MoreResources.readResource(DataArgumentsProvider.EXCHANGE_RATE_CONFIG.getCatalogFileVersion(getProtocolVersion())),
             AirbyteCatalog.class);
-    final String namespace = "sourcenamespace";
+    // A randomized namespace is required otherwise you can generate a "false success" with data from a
+    // previous run.
+    final String namespace = Strings.addRandomSuffix("airbyte_source_namespace", "_", 8);
```

Comment:
do we have cleanup logic for these randomized datasets? Or is the plan to just add a TTL on the tables (or some other way to clean them out eventually)

(it looks like `tearDownBigQuery` only deletes the default dataset, so these `airbyte_source_namespace_*` won't get deleted)